packages/apidom-ls/src/config/openapi/path-item/documentation.ts

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
Missing `This object MAY be extended with [Specification Extensions](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#specificationExtensions)` I guess..

Reply:
fixed